var/spack/repos/builtin/packages/mgis/package.py

```diff
@@ -104,4 +105,7 @@ def cmake_args(self):
             # adding path to boost
             args.append("-DBOOST_ROOT={0}".format(self.spec["boost"].prefix))
 
+        if "+static" in self.spec:
+            args.append("-Denable-static=ON")
+
```

Comment:
FYI. You could use the `self.define_from_variant()` helper here if you want.  See https://spack.readthedocs.io/en/latest/build_systems/cmakepackage.html#adding-flags-to-cmake.

Reply:
Didn't know, thanks @tldahlgren. 

@thelfer I let you make the final decision 😉

Reply:
Ok for me